### PR TITLE
no reason to store user, only user id

### PR DESCRIPTION
**Pull request by itayd**

### CODE REVIEW

**Review by ig-ra — APPROVED** [review]

File: internal/backend/db/dbgorm/ownership.go [hunk 1]
```diff
@@ -154,72 +145,72 @@ func isUserEntity(db *gorm.DB, user *scheme.User, ids ...sdktypes.UUID) error {
...
+	return joinUserEntity(db, entity, uid)
 }
 
 type PermissiveOwnershipChecker struct {
 	z *zap.Logger
 }
 
-func (c *PermissiveOwnershipChecker) IsUserEntity(db *gorm.DB, user *scheme.User, ids ...sdktypes.UUID) error {
+func (c *PermissiveOwnershipChecker) IsUserEntity(db *gorm.DB, uid string, ids ...sdktypes.UUID) error {
 	return nil
```
Comment (tebeka) [inline internal/backend/db/dbgorm/ownership.go:211]: What is this function?

**Review by tebeka — APPROVED** [review]
One minor question

File: internal/backend/db/dbgorm/ownership.go (hunk 1 above)
Comment (itayd) [reply internal/backend/db/dbgorm/ownership.go:211]: @ig-ra i'm not sure?

Comment (ig-ra) [reply internal/backend/db/dbgorm/ownership.go:211]: function that returns whether provided IDs are belong/accessible by the given user

**Pull request merged**